docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -79,33 +79,102 @@ features, like:
 Note that data modification also requires implementing
 a :ref:`connector-page-sink-provider`.
 
+When a ``SELECT`` query is being executed, and it contains specific clauses,
```

Comment:
Probably add a heading for `Pushdown`.

Reply:
Can we add it when there's going to be at least one other type of pushdown documented?